packages/salesforce-adapter/src/change_validators/flows.ts

```
@@ -263,6 +270,9 @@ const activeFlowValidator =
       .filter(isActivatingChange)
       .map(change => {
         if (isActivatingChangeOnly(change)) {
+          if (getFlowStatus(change.data.before) === INVALID_DRAFT) {
```

are you sure you want to throw this error only if its activating change only? 
I think that we want to error this even if the user did other changes, so this needs to be outside this if

---

I agree, done.